Symfony/CS/Fixer/Contrib/ModernizeTypesCastingFixer.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Functions\FunctionArgumentsUtil;
+use Symfony\CS\Functions\FunctionDefinitionUtil;
+use Symfony\CS\Functions\FunctionReference\Finder;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ */
+final class ModernizeTypesCastingFixer extends AbstractFixer
```

Comment:
You should override `getPriority` method and add new test inside https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.10/Symfony/CS/Tests/FixerTest.php#L158